roles/ceph-config/tasks/main.yml

```diff
@@ -9,6 +9,24 @@
       group: "ceph"
       mode: "0755"
 
+  - name: count number of osds for non-lvm scenario
+    set_fact:
+      num_osds: "{{ devices | length }}"
+    when:
+      - devices|length > 0 and (osd_scenario == 'collocated' or osd_scenario == 'non-collocated')
```

Comment:
``(osd_scenario == 'collocated' or osd_scenario == 'non-collocated')`` should be on a new line

```
when:
  - devices|length > 0
  - (osd_scenario == 'collocated' or osd_scenario == 'non-collocated')
```

Reply:
I used `and` based on https://github.com/neha-ojha/ceph-ansible/commit/9e203db2334ee04449451e240473860e4749d454#r30481853. I'll put them in two lines.

Reply:
I used and based on neha-ojha@9e203db#r30481853. I'll put them in two lines.